.appveyor.yml

```diff
@@ -1,24 +1,85 @@
-## Operating System (VM environment) ##
-platform: x64
-image: Visual Studio 2017
-
-## Install Script ##
-install:
-  - set CYGWIN_ROOT=C:\cygwin64
-  - set PATH=%CYGWIN_ROOT%\bin;%PATH%
-  - bash --version
-  - bash -c 'autoreconf --version'
-  - bash -c 'make --version'
-
-## Disable MSBuild (default) ##
-build: off
-
-## Build Script ##
+version: 3.1.6-{build}
+
+image:
+  - Visual Studio 2013
+  - Visual Studio 2017
+
+clone_depth: 1
+
+before_build:
+  - ps: >-
+      if($env:PLATFORM -like "cygwin*") {
+        $env:CYGWIN_ROOT="C:\$env:PLATFORM"
+        $env:PATH="$env:CYGWIN_ROOT\bin;$env:PATH"
+      } else {
+        cd C:\projects\libmodbus\src\win32
```

Comment:
Same here

Reply:
The paths in that position are dependent on AppVeyor's build image, on which the whole .appveyor.yml is dependent. If the build image changes, chances are high it won't work anymore, anyways. But I guess it can be replaced by an env var.